apps/communications/dialer/index.html

```diff
@@ -36,6 +36,7 @@
 
     <link rel="stylesheet" type="text/css" href="app://theme.gaiamobile.org/shared/elements/gaia-theme/gaia-theme.css">
     <link rel="stylesheet" type="text/css" href="/shared/elements/gaia-icons/gaia-icons.css">
+    <link rel="stylesheet" type="text/css" href="/shared/elements/gaia-icons/bidi-helper.css">
```

Comment:
I guess I need to ask a review from a communication peer. They might not want a full css just for one icon...